client/blocks/inline-help/inline-help-forum-guide.jsx

```diff
@@ -0,0 +1,39 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import Button from 'components/button';
+import { preventWidows } from 'lib/formatting';
+
+const FORUM_LINK = '//en.forums.wordpress.com';
+
+const InlineHelpRichForumGuide = ( { translate = noop } ) => (
+	<div className="inline-help__forum-view">
+		<p>
+			{ preventWidows(
+				translate(
+					'Post a new question in our {{strong}}public forums{{/strong}}, ' +
+						'where it may be answered by helpful community members, ' +
+						'by following the link below.',
+					{
+						components: {
+							strong: <strong />,
+						},
+					}
+				)
+			) }
+		</p>
+		<Button href={ FORUM_LINK } target="_blank" rel="noopener noreferrer" primary>
+			{ translate( 'Support Forums' ) }
```

Comment:
Optional: Add an action at the beginning maybe? `Go to` or `Open`

Reply:
I'm certainly not against any of those suggestions :)
I'll request a copy review for the sake of the description copy and see what our copy crew think of this.